sui/sources/movescription.move

```diff
@@ -537,6 +538,17 @@ module smartinscription::movescription {
         let coin_bag: &mut Bag = dof::borrow_mut<vector<u8>, Bag>(&mut inscription.id, COIN_BAG);
         let coin_type: TypeName = type_name::get<T>();
         bag::add(coin_bag, coin_type, receive);
+
+    // ===== Migrate functions =====
+
+    entry fun migrate_deploy_record(deploy_record: &mut DeployRecord) {
```

Comment:
It would be better to require some Admin Cap here. Or older version can call this again.